graviti/schema/builtin.py

```diff
@@ -273,7 +297,13 @@ def __init__(
     ) -> None:
         super().__init__(fields=fields, nullable=nullable)
 
-    def _get_pyarrow_type(self) -> "paDataType":
+    def to_pyarrow(self) -> pa.DataType:
+        """Convert the Portex type to the corresponding builtin PyArrow DataType.
+
+        Returns:
+            The corresponding builtin PyArrow DataType.
+
+        """
         return pa.struct(self.fields.to_pyarrow())  # pylint: disable=no-member
```

Comment:
where the pylint warning comes from?

Reply:
Pylint considers `self.fields` as a tuple, which does not have `to_pyarrow` method.